docker/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Correcting the URL format
+URL=$(echo "$URL" | sed 's|/$||')  # Remove trailing slash if exists
+if ! echo "$URL" | grep -q "^https://"; then
+  URL="https://$URL"  # Add https:// if not present
+fi
+
+# Exporting the corrected URL
+export URL
+
+# Run pikaraoke with environment variables
+if [ -z "$PASSWORD" ]; then
+  figlet PiKaraoke
```

Comment:
Minor nitpick, but  figlet is being run in both conditions, so you can remove it and place one just above the if block

Reply:
Oh! haha... not sure how I missed that.